src/lib.rs

```diff
@@ -506,6 +506,19 @@ pub fn script_num_size(n: usize) -> usize {
     }
 }
 
+/// FIXME
```

Comment:
I need some help here. I don't totally understand what this does!

Reply:
This should be renamed to `push_opcode_size` (and arguably should live in rust-bitcoin rather than here, but that's a problem for another day). An explanation and doccomment might be
```
/// Returns the size of the smallest push opcode used to push a given number of bytes onto the stack
///
/// For sizes ≤ 75, there are dedicated single-byte opcodes, so the push size is one. Otherwise, if the
/// size can fit into 1, 2 or 4 bytes, we use the `PUSHDATA{1,2,4}` opcode respectively, followed
/// by the actual size, encoded in that many bytes.
```

Reply:
Let's also not make this `pub`, since we'll want to remove it if/when it moves into rust-bitcoin

Reply:
Removed `pub`